### PR TITLE
Run tests on develop

### PR DESCRIPTION
**Pull request by jmaslek**
^^

### CODE REVIEW

**Review by deeleeramone — APPROVED** [review]
LGTM

File: .github/workflows/platform-core.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
         fail-fast: false
         matrix:
           python_version:
-            ["3.8", "3.9", "3.10", "3.11"]
```
Comment (hjoaquim) [inline .github/workflows/platform-core.yml:19]: Are we dropping 3.8 support??

**Comment by piiq** [comment]
you think we can make a condition to run the tests only if the diff only has files in openbb_platform? or we have that already

File: .github/workflows/platform-core.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ on:
   pull_request:
     branches:
       - develop
+    paths:
+      - 'openbb_platform/**'
```
Comment (piiq) [inline .github/workflows/platform-core.yml:8]: That was fast

**Review by piiq — APPROVED** [review]

**Pull request merged**